tests/api_service_test.py

```diff
@@ -0,0 +1,16 @@
+from vcr_unittest import VCRTestCase
+from api_service import ApiService
+
+class TestApiService(VCRTestCase):
+
+    def test_create_desk_is_successful(self):
+        response = ApiService.create_desk(name = 'test desk')
+        json_response = response.json()
+
+        self.assertEqual(response.status_code, 201)
+        assert 'id' in json_response
```

Comment:
`self.assertTrue`

[Why not to use assert.](https://stackoverflow.com/questions/6361147/why-do-assertions-in-unittest-use-testcase-assertequal-not-the-assert-keyword)

Reply:
Here I only want to test that the key exists in the `json_response` dictionary, regardless of what it is so `assertTrue` doesn't seem to be what I'm looking for.

What's on your mind?

Reply:
`assert` is not as reliable as `self.assertTrue` because it can be disabled, as unlikely as that is. I think best practice is to use the latter over the first. My suggestion is to write this instead:
```
self.assertTrue('id' in json_response)
```